spec/factories/interviews.rb

```diff
@@ -2,9 +2,9 @@
 
 FactoryBot.define do
   factory :interview do
-    schedule { '2020-01-27 19:16:51' }
-    attendable { false }
-    interviewer_id { 1 }
-    interviewee_id { 1 }
+    user_id { FactoryBot.create(:user, :member).id }
+    interviewer_id { FactoryBot.create(:user, :interviewer).id }
```

Comment:
```suggestion
    user { FactoryBot.create(:user, :member) }
    interviewer { FactoryBot.create(:user, :interviewer) }
```

こう書けた気がする（これは github の suggestions という機能を使ってて、そのまま取り込めるます）